src/app/components/challenge-create/challenge-create.component.scss

```diff
@@ -12,10 +12,10 @@
 }
 
 .web-container {
-  width: calc(100vw - 223px);
-  float: right;
   padding-top: 70px;
   overflow-x: hidden;
+  width: calc(100vw - 223px);
+  float: right;
```

Comment:
Please revert all the changes of this file.

Reply:
Then how will the challenge-flex divs be styled?

Reply:
No actually these changes were already there na on line no 15 and 16, so let it be there only.

Reply:
I'm confused here on what you want me to do

Reply:
Just make this file as we have in the master branch of this repo so that this file doesn't contain any change we already have the same width and float value.